talk/tools/compiling.tex

```diff
@@ -64,11 +64,11 @@
   \frametitle{Useful compiler options (gcc/clang)}
   \begin{block}{Get more warnings}
     \begin{description}
-      \item[-Wall -Wextra] the way to get all warnings
-      \item[-Werror] the way to force yourself to look at warnings
+      \item[-Wall -Wextra] get all warnings
+      \item[-Werror] force yourself to look at warnings
     \end{description}
   \end{block}
-  \begin{block}{Around optimization}
+  \begin{block}{Optimization}
     \begin{description}
       \item[-g] add debug symbols
       \item[-O0, -O2] 0 = no optimization, -O2 = optimized
```

Comment:
I wonder why we do not list `-O3` here. Until very recently you would not get vectorization with `-O2` so it is an important flag for my use cases.

Reply:
I mention `-O3` and `-Og` now.